services/deployment-agent/deployment_config.default.yaml

```diff
@@ -47,7 +47,7 @@ main:
             cat .env-devel >> .env && cat .env.platform >> .env &&
             docker-compose -f services/docker-compose.yml -f docker-compose.deploy.yml config > stack.yml
     stack_file: stack.yml # the output file of the command above, or just the file to use
-    excluded_services: [minio]
+    excluded_services: [webclient, minio]
```

Comment:
MINOR: there is no more webclient, right? In any case, it does not harm to have it... this way you can deploy old versions of simcore

Reply:
yes currently the production cluster still has the webclient in.